src/Elcodi/Component/Tax/Tests/UnitTest/Entity/TaxTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/*
+ * This file is part of the Elcodi package.
+ *
+ * Copyright (c) 2014-2015 Elcodi Networks S.L.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Feel free to edit as you please, and have fun.
+ *
+ * @author Marc Morera <yuhu@mmoreram.com>
+ * @author Aldo Chiecchia <zimage@tiscali.it>
+ * @author Elcodi Team <tech@elcodi.com>
+ */
+
+namespace Elcodi\Component\Tax\Tests\UnitTest\Entity;
+
+use Elcodi\Component\Core\Tests\UnitTest\Entity\AbstractEntityTest;
+use Elcodi\Component\Core\Tests\UnitTest\Entity\Traits;
+use Elcodi\Component\Tax\Entity\Tax;
+
+class TaxTest extends AbstractEntityTest
+{
+    use Traits\IdentifiableTrait, Traits\EnabledTrait;
+
+    /**
+     * Sets up the fixture, for example, opens a network connection.
+     * This method is called before a test is executed.
+     */
+    protected function setUp()
```

Comment:
I've used it to the the `__toString`, should it be included in the abstract system?